src/chef-mover/rebar.config

```diff
@@ -112,6 +112,7 @@
                     {platform_define, "^17.", 'ERLANG_OTP_VERSION_17'},
                     {platform_define, "^18.", 'ERLANG_OTP_VERSION_18'},
                     {platform_define, "^19.", 'ERLANG_OTP_VERSION_19'},
+                    {platform_define, "^20.", 'ERLANG_OTP_VERSION_20'},
```

Comment:
Do not see this being used anywhere in the code.

Reply:
What is the correct change here?

Reply:
Lets leave this as is for now - might be something to consider for the future.